docs/ssm-deployment-guide.md

```diff
@@ -485,7 +487,7 @@ If the current configuration cannot meet your performance requirements, you can
 ```
 ## Cmdlet history purge in metastore  
 
-SSM choose to save cmdlet and action execution history in metastore for audit and log purpose. To not blow up the metastore space, SSM support periodically purge cmdlet and action execution history. Property `smart.cmdlet.hist.max.num.records` and `smart.cmdlet.hist.max.record.lifetime` are supported in smart-site.xml.  When either condition is met, SSM will trigger backend thread to purge the history records.
+SSM choose to save cmdlet and action execution history in metastore for audit and log purposes. To not blow up the metastore space, SSM support periodically purging cmdlet and action execution history. Property `smart.cmdlet.hist.max.num.records` and `smart.cmdlet.hist.max.record.lifetime` are supported in smart-site.xml.  When either condition is met, SSM will trigger backend thread to purge the history records.
```

Comment:
SSM will save ..

Reply:
To avoid blowing up the metastore space, SSM supports configuration for periodically purging historical records for cmdlets and actions.